src/components/AddAnimal.vue

```diff
@@ -42,25 +69,42 @@ export default {
   data: () => ({
     dialog: false,
     AnimalName: "",
+    fileInputKey: 0,
+    image: null,
+    snackbar: "",
+    AddedText: "Animal card is successfully added",
+    Added_card: false,
   }),
 
   methods: {
     ...mapActions("Anima", ["AddItems"]),
 
     addAnimal() {
-      console.log("egtsrh");
-
-      this.addItems({
-        imgSrc: 154584,
-        name: this.AnimalName,
-        //   id: this.items.length + 1,
-      });
-
-      this.AnimalName = "";
+      if (this.image != null && this.AnimalName.trim().length > 0) {
```

Comment:
const removedSpace = this.AnimalName.trim()
if (this.image  && removedSpace), this will also work here. Now also look for code optimizing.